### PR TITLE
feat: Added sharing functionality to Update Event

### LINKED ISSUE #1048: Add the sharing functionality in Update Event toolbar
### Feature Request

**Feature Description**
Eventbrite has a sharing functionality at the toolbar and it should be implemented in the orga app as well.

**Would you like to work on the issue?**
Yes!

**Screenshot**
ezgif com-resize 2

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #1048 

Screenshots for the change: 
whatsapp_image_2018-06-10_at_23 15 11_50

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Fix codacy issue

**Comment by iamareebjamal** [comment]
Also, commit guidelines about imperative messages are not being followed

**Comment by sridharjajoo** [comment]
@iamareebjamal I have done the required change. Hope its fine now.

File: app/src/main/java/org/fossasia/openevent/app/core/event/create/CreateEventPresenter.java [hunk 1]
```diff
@@ -158,4 +158,20 @@ void onPaymentCountrySelected(String paymentCountry) {
...
+            .append("Starts: ").append(DateUtils.formatDateWithDefault(DateUtils.FORMAT_DAY_COMPLETE, event.getStartsAt()))
+            .append(doubleLineBreak)
+            .append("Ends at: ").append(DateUtils.formatDateWithDefault(DateUtils.FORMAT_DAY_COMPLETE, event.getEndsAt()));
+
+        if (event.getExternalEventUrl() != null) {
+            data.append(doubleLineBreak).append("Url: ").append(event.getExternalEventUrl());
+        }
+
+        return data.toString();
+    }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/event/create/CreateEventPresenter.java]: This seems to be repeating. Create common method in Utils

File: app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventPresenter.java [hunk 2]
```diff
@@ -117,21 +117,7 @@ private void listenChanges() {
...
-            .append(doubleLineBreak)
-            .append("Ends at: ").append(DateUtils.formatDateWithDefault(DateUtils.FORMAT_DAY_COMPLETE, event.getEndsAt()));
-
-        if (event.getExternalEventUrl() != null) {
-           data.append(doubleLineBreak).append("Url: ").append(event.getExternalEventUrl());
-        }
-
-        return data.toString();
-    }
+    public Event getEvent() {return event;}
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventPresenter.java]: Why not call the Util method here itself

**Comment by sridharjajoo** [comment]
@iamareebjamal Kindly review

File: app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java [hunk 3]
```diff
@@ -25,7 +25,6 @@
 import org.fossasia.openevent.app.data.event.Event;
 import org.fossasia.openevent.app.databinding.AboutEventFragmentBinding;
 import org.fossasia.openevent.app.ui.ViewUtils;
-
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java]: Unneeded change. Kindly check the change list after updating PR to save time in review request cycles

**Comment by sridharjajoo** [comment]
I will take care about it from next time.Done the changes.

**Comment by iamareebjamal** [comment]
@sridharjajoo Please follow commit message guidelines

**Pull request merged**